isthmus/src/test/java/io/substrait/isthmus/RoundingFunctionTest.java

```diff
@@ -0,0 +1,33 @@
+package io.substrait.isthmus;
+
+import java.util.List;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class RoundingFunctionTest extends PlanTestBase {
+
+  static List<String> CREATES =
+      List.of(
+          "CREATE TABLE numbers (i8 TINYINT, i16 SMALLINT, i32 INT, i64 BIGINT, fp32 FLOAT, fp64 DOUBLE, decimal DECIMAL(5,2)");
+
+  @ParameterizedTest
+  @ValueSource(strings = {"fp32", "fp64"})
+  void ceil(String column) throws Exception {
+    String query = String.format("SELECT ceil(%s) FROM numbers", column);
+    assertSqlSubstraitRelRoundTrip(query, CREATES);
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"fp32", "fp64"})
+  void floor(String column) throws Exception {
+    String query = String.format("SELECT floor(%s) FROM numbers", column);
+    assertSqlSubstraitRelRoundTrip(query, CREATES);
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"decimal"})
+  void round(String column) throws Exception {
+    String query = String.format("SELECT round(%s, 1) FROM numbers", column);
+    assertSqlSubstraitRelRoundTrip(query, CREATES);
```

Comment:
The version of substrait that's submoduled in substrait-java doesn't have `round` defined.

https://github.com/substrait-io/substrait/blob/6a74f40f0d835a447a7175b20dd430c7c39c5d11/extensions/functions_rounding.yaml